code/Jen/javascript/Lab_06/quoteAPI.js

```diff
@@ -0,0 +1,68 @@
+
+
+const App = {
+
+    el: '#app',
+    data() {
+        return {
+            output: {},
```

Comment:
tip: if you make these null instead of empty object, it will fix the weirdness with the input field